devices/src/bus.rs

```diff
@@ -29,6 +30,18 @@ pub trait BusDevice: AsAny + Send {
     fn interrupt(&self, irq_mask: u32) {}
 }
 
+/// Trait for devices that handle raw non-blocking I/O requests.
```

Comment:
nit: typo in commit "Rework serial device integration in VMM": s/`implmenet`/`implement`

Same commit message says:
> Rename LegacyDeviceManager to PortIODeviceManager

But the renaming has been done in the previous commit.

Reply:
I'll move this commit message to where it belongs.